components/google-cloud/google_cloud_pipeline_components/experimental/evaluation/component.yaml

```diff
@@ -0,0 +1,57 @@
+name: Launch Evaluation Dataflow Flex Template
```

Comment:
Just a heads up @SinaChavoshi is refreshing the dataflow flex template component so the interface might change sometime soon in the future. 
You could still check in this to test the functionality and we can refresh the YAML later.